src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -65,14 +87,25 @@ export class SavedObjectsClient {
     }
   }
 
+  /**
+   * @param {object} options
+   * @param {string} options.type
```

Comment:
LIke above, properties of an object param should be doc'd as `@property {type} options.name`